R/get_inboveg_ppa.R

```diff
@@ -0,0 +1,294 @@
+        user_reference = user_reference,
+        recording_givid = recording_givid,
+        .con = connection)
+    }
+    if ((length(survey_name) == 1 && survey_name[1] == "%") &&
+        !(length(user_reference) == 1 && user_reference[1] == "%") &&
+        !(length(recording_givid) == 1 && recording_givid[1] == "%")) {
+      sql_statement <- glue_sql(
+        common_part,
+        "AND (ivR.[RecordingGivid] IN ({recording_givid*})
+         OR ivR.UserReference IN ({user_reference*}))",
+        user_reference = user_reference,
+        recording_givid = recording_givid,
+        .con = connection)
+    }
+    if ((length(survey_name) == 1 && survey_name[1] == "%") &&
+        (length(user_reference) == 1 && user_reference[1] == "%") &&
+        !(length(recording_givid) == 1 && recording_givid[1] == "%")) {
+      sql_statement <- glue_sql(
+        common_part,
+        "AND ivR.[RecordingGivid] IN ({recording_givid*})",
+        recording_givid = recording_givid,
+        .con = connection)
+    }
+    if ((length(survey_name) == 1 && survey_name[1] == "%") &&
+        !(length(user_reference) == 1 && user_reference[1] == "%") &&
+        (length(recording_givid) == 1 && recording_givid[1] == "%")) {
+      sql_statement <- glue_sql(
+        common_part,
+        "AND ivR.UserReference IN ({user_reference*})",
+        user_reference = user_reference,
+        .con = connection)
+    }
+    if (!(length(survey_name) == 1 && survey_name[1] == "%") &&
+        (length(user_reference) == 1 && user_reference[1] == "%") &&
+        (length(recording_givid) == 1 && recording_givid[1] == "%")) {
+      sql_statement <- glue_sql(
+        common_part,
+        "AND ivS.Name IN ({survey_name*})",
+        survey_name = survey_name,
+        .con = connection)
+    }
+    if (!(length(survey_name) == 1 && survey_name[1] == "%") &&
+        (length(user_reference) == 1 && user_reference[1] == "%") &&
+        !(length(recording_givid) == 1 && recording_givid[1] == "%")) {
+      sql_statement <- glue_sql(
+        common_part,
+        "AND ivS.Name IN ({survey_name*})
+         AND ivR.[RecordingGivid] IN ({recording_givid*})",
+        survey_name = survey_name,
+        recording_givid = recording_givid,
+        .con = connection)
+    }
+    if (!(length(survey_name) == 1 && survey_name[1] == "%") &&
+        !(length(user_reference) == 1 && user_reference[1] == "%") &&
+        (length(recording_givid) == 1 && recording_givid[1] == "%")) {
+      sql_statement <- glue_sql(
+        common_part,
+        "AND ivS.Name IN ({survey_name*})
+         AND ivR.UserReference IN ({user_reference*})",
+        survey_name = survey_name,
+        user_reference = user_reference,
+        .con = connection)
+    }
+    if (!exists("sql_statement")) {
+      stop("When choosing multiple = TRUE, please add a (vector of) value(s) for at least one of the following variables: survey_name, user_reference or recording_givid.") #nolint
+    }
```

Comment:
```suggestion
    stopifnot(
      "When choosing multiple = TRUE, please add a (vector of) value(s) for at least one of the following variables: survey_name, user_reference or recording_givid." #nolint
      = exists("sql_statement")
    )
```
And another trick to decrease the cyclcomp.  ;-)